demos/chatroom/experiment.py

```diff
@@ -14,12 +18,16 @@ def __init__(self, session):
         self.initial_recruitment_size = self.num_participants
         self.quorum = self.num_participants
         self.setup()
+        self.config = dlgr.config.get_config()
+        self.config.register('network', unicode)
+        if not self.config.ready:
+            self.config.load_config()
```

Comment:
Won't the config usually have been loaded (without the network parameter registered) before we get here? I think we may have to do more work on how we handle local experiment configuration, but I'm still somewhat confused myself about how it should work.